src/pages/interviews/realworldreact

```diff
@@ -0,0 +1,127 @@
+
+[Real World React – Sep 22, 2022](https://www.youtube.com/watch?v=NAbEo2_6Us4&ab_channel=RealWorldReact)
+
+## Talking Points
+
+- What's so great about being a senior engineer?
+- Characteristics of a Senior Engineer
+- Senior Engineer vs People Manager
+- Senior Engineer vs Lead Engineer
+- Do senior engineers follow on these responsibilities of product features and design versus product development?
+- The Senior Engineer title
+- Do engineers report to a senior engineer?
+- Having a vision for your career
+- How meetings give you superpowers
+- The mindset of the senior engineer
+- Is leadership a quality that is a requisite for becoming a senior engineer?
+-  Visas and working in the Bay
+- Blogging and how it helps your career
+- Working with different tech stacks
+- Choosing a company to join
+- Gathering consensus among engineers 
+- Best practises for a junior developer to implement now to become a senior developer one day
+- Dealing with trials, tribulations and challenges
+
+
+## Quotable Quotes
+
+"My favorite thing about being a software engineer is that it makes it very easy to go from an idea to something that actually works and does things for people." - Swizec Teller
+
+"A senior software engineer should be able to take a fuzzy description of a problem and find a solution for that and implement it, adapt it to your system" - Swizec Teller
+
+"Junior or associate engineers usually are people who can write code. They can write really good code, but it's not so much about how long it takes them, but they need more guidance and more guardrails" - Swizec Teller
+
+"Senior engineers and then senior plus and above, that's where you start getting into the area where people are like somewhere between purely focused on the code and starting to become more focused on the business problem" - Swizec Teller
+
+"A very typical differentiator between someone who's like a mid-level or a senior engineer is whether they can fit the solution into the constraints" - Swizec Teller
+
+"Titles are always given, but it's kind of taken in that managers usually look at the people they have on the team and they see who has the potential" - Swizec Teller
+
+"Senior engineers can kind of take on the role of a tech lead on a team or leading a project that potentially even spans multiple teams, but they're not really people managers. It's more like managing the technical aspects, maybe running the project management part depending on company size and culture" - Swizec Teller
+
+"As a senior engineer, you're often owning the project and responsible for its delivery even though you're not the person who is writing all of the code." - Swizec Teller
+
+"Senior engineer is more like project focused, making sure a project get done well and on time rather than constructing a team" - Swizec Teller
```

Comment:
```suggestion
"Senior engineer is more like project focused, making sure a project gets done well and on time rather than constructing a team" - Swizec Teller
```